linz-data-importer/about.html

```diff
@@ -0,0 +1,27 @@
+<h1 id="linz-data-importer">LINZ Data Importer</h1>
+<p>This plugin allows you to easily discover and import data from Toitū Te Whenua Land Information New Zealand into QGIS.</p>
+<p>Explore and connect to LINZ's WMTS and WFS services from within QGIS.</p>
+
+<h2>Available data</h2>
+<p>Data (available through WFS or WMTS) from the agency data portals and services below are available via the LINZ Data Importer.</p>
+
+<h3>Open data portals</h3>
+<ul>
+    <li>Toitū Te Whenua – Land Information New Zealand (<a href="https://data.linz.govt.nz">https://data.linz.govt.nz</a>)</li>
```

Comment:
Could we simplify these links to use the name of the target rather than the URL as the link content? That is, `<li><a href="https://data.linz.govt.nz">Toitū Te Whenua – Land Information New Zealand</a></li>`.